js/signin.js

```diff
@@ -1,72 +1,79 @@
-const emailInput = document.querySelector("#email-input");
-const passwordInput = document.querySelector("#password-input");
-const emailErrorMessage = document.querySelector("#email-error-message");
-const passwordErrorMessage = document.querySelector("#password-error-message");
-const loginForm = document.querySelector("#login-form");
+const querySelector = (selector) => document.querySelector(selector);
 
-const eyeOffButton = document.querySelector(".eye-off");
-const eyeOnButton = document.querySelector(".eye-on");
+const emailInput = querySelector("#email-input");
+const passwordInput = querySelector("#password-input");
+const emailErrorMessage = querySelector("#email-error-message");
+const passwordErrorMessage = querySelector("#password-error-message");
+const loginForm = querySelector("#login-form");
+
+const eyeOffButton = querySelector("#eye-off");
+const eyeOnButton = querySelector("#eye-on");
 
 const EMAIL_REGEX = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,6}$/;
 
 // 이메일 입력
-emailInput.addEventListener("focusout", checkEmailInput);
-emailInput.addEventListener("focus", resetInputStyle);
+emailInput.addEventListener("focusout", validateEmailInput);
+
+function validateEmailInput() {
+  const email = emailInput.value.trim();
 
-function checkEmailInput() {
-  if (emailInput.value.trim() === "") {
+  if (email === "") {
     showEmailErrorMessage("이메일을 입력해 주세요.");
-  } else if (!validateEmail(emailInput.value.trim())) {
+    return;
+  }
+
+  if (!isValidateEmail(email)) {
     showEmailErrorMessage("올바른 이메일 주소가 아닙니다.");
-  } else {
-    hideErrorMessage();
+    return;
   }
+
+  hideEmailErrorMessage();
 }
 
-function validateEmail(email) {
+function isValidateEmail(email) {
```

Comment:
validate는 동사의 의미를 가지고 있어서 valid로 표현해주시면 좋을 것 같아요!
유효하지 않은 경우에는 invalid라고 쓰는 편이에요!
```suggestion
function isValidEmail(email) {
```